src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3087,6 +3127,12 @@ namespace winrt::TerminalApp::implementation
             case SettingsTarget::SettingsFile:
                 openFile(CascadiaSettings::SettingsPath());
                 break;
+            case SettingsTarget::FileExplorer:
+                openFolder(CascadiaSettings::SettingsDirectory());
+                break;
+            case SettingsTarget::Clipboard:
+                copyToClipboard(CascadiaSettings::SettingsPath().c_str());
```

Comment:
The `.c_str()` is unnecessary, if not detrimental, because constructing a string-view from a nullptr is UB.

Reply:
Updated.  (Sorry,  I should have thought that through)

Let me know if I should remove the other 2 actions.  Fwiw, the send input one is what I have found most useful, I don't think I have ever actually used the other 2 outside of testing.  But that is probably just because of my personal workflows.